app/routes/circulars/index.tsx

```diff
@@ -206,6 +211,16 @@ export default function () {
           </Button>
         </Link>
       </ButtonGroup>
+      <p className="usa-hint">
+        <Hint>
+          Search for Circulars by submitter, subject, or body text. For example,
+          to search for GRB circulars from Fermi, search for 'Fermi GRB'.
+        </Hint>
+        <Hint>
+          To navigate to a specific circular, search for the associated Circular
+          ID ('gcn123', 'Circular 123', '123', etc.)
+        </Hint>
+      </p>
```

Comment:
- React doesn't like `<p>` tags to contain `<div>`s. (The `<Hint>` component has a `<div>` inside it, so a `<p>` cannot contain a `<Hint>`.) See screen shot below showing the client-side error from React.
- Add an `id` attribute so that we can link the input field to the hint using Aria attributes.

![Screenshot 2023-05-25 at 14 43 47](https://github.com/nasa-gcn/gcn.nasa.gov/assets/728407/cf7f0860-357f-4b8c-8d83-601feb0faa8f)

```suggestion
        <Hint id="searchHint">
          Search for Circulars by submitter, subject, or body text. For example,
          to search for GRB circulars from Fermi, search for 'Fermi GRB'.<br/>
          To navigate to a specific circular, search for the associated Circular
          ID ('gcn123', 'Circular 123', '123', etc.)
        </Hint>
```

Reply:
ah, my apologies, I missed that